### DISCUSSION TITLE
[BE] 방문 횟수 로직 변경 시 데이터 migration 이슈

### DISCUSSION THREAD

**Discussion by yenawee** [woowacourse-teams/2023-stamp-crush#397] [General]
방문 횟수 로직 구하는 방법 확실히 정하기
데이터 마이그레이션 방법 정하기

**Comment by yenawee** [comment]
## 레오의 현재 로직
- 테이블 구조를 혼자 바꿀 수 없어서 이렇게 구현함

- stamp의 createdAt 을 분단위까지 distinct 를 해서 개수를 구함
- 허점이 있다 

https://github.com/woowacourse-teams/2023-stamp-crush/discussions/173
<-
VISIT_HISTORY 테이블 만들자고 결정한 이슈

**Comment by gitchannn** [comment]
위에서 결정된 내용에 따라서 신규 테이블을 생성하기로 함. 
기존에 방문한 횟수에 대해서는 어떻게 해야할까?

**Comment by gitchannn** [comment]
# 결정된 내용

## 추가될 테이블 구조

- stamp_count 추가: 한 방문 건에서 추가된 스탬프 개수 

```
VISIT HISTORY
cafe_id
customer_id
stamp_count 
created_at
updated_at
```

## 데이터 Migration 관련

- flyway를 지금 공부해서 하기는 너무 어렵다. 
- 방문횟수 데이터는 현재 로직에서 구해지는 방문횟수 넣어두기
- 일단 수작업으로 진행한다. 
- 스탬프 적립할때 data insert
